dsgrid_project/dimensions/subset/end_uses_by_sector.csv

```diff
@@ -0,0 +1,63 @@
+id,commercial_end_uses,industrial_end_uses,residential_end_uses,transportation_end_uses
+district_cooling_cooling,x,,,
+district_heating_heating,x,,,
+district_heating_water_systems,x,,,
+electricity_bath_fan,,,x,
+electricity_ceiling_fan,,,x,
+electricity_clothes_dryer,,,x,
+electricity_clothes_washer,,,x,
+electricity_cooking_range,,,x,
+electricity_cooling,x,,x,
+electricity_dishwasher,,,x,
+electricity_ev_dcfc,,,,x
+electricity_ev_l1l2,,,,x
+electricity_exterior_holiday_lighting,,,x,
+electricity_exterior_lighting,x,,x,
```

Comment:
I just realized that when one base dimension record gets mapped to multiple subsets:

1. That's totally appropriate here, but
2. It makes the supplemental dimension created from the column names somewhat ambiguous. 

Are we checking for and/or doing anything about this condition?

Reply:
No. Why are the columns ambiguous?

Reply:
The columns themselves aren't ambiguous, but if the project data is mapped to the supplemental dimension defined by the columns, some of the base dimension data will be double-counted. In this particular case, electricity_exterior_lighting energy will show up in both commercial_end_uses and residential_end_uses, and could thus be double-counted.

Reply:
I raised this in a Teams discussion a while back and the conclusion was that this case is OK because the electricity_exterior_lighting would be in different rows for commercial vs residential, and so wouldn’t be double counted. Did we miss something?

Reply:
I think we did miss something, because I think for `com` rows we'd end up with all energy use under `commercial_end_uses` and a subset of the energy use under `residential_end_uses`, and vice-versa for `res` rows.